src/main/java/seedu/address/model/Model.java

```diff
@@ -103,5 +104,30 @@ public interface Model {
      */
     Person getDeletedPerson();
 
-    void undo();
+    /**
+     * Removes the most recently deleted person from the deleted person arraylist.
+     */
+    void removeDeletedPerson();
+
```

Comment:
Should add javadoc comments for all the methods below

Reply:
Added!